FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Logging/IDiagnosticLogger.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Logging
+{
+    public interface IDiagnosticLogger
+    {
+        public void LogInformation(string message);
+
+        public void LogWarning(string message);
+
+        public void LogError(string errorMessage);
```

Comment:
Would it be better to add exception parameter in LogError?

Reply:
we can add it in the future if needed.